mage_ai/services/spark/spark.py

```diff
@@ -1,8 +1,42 @@
 from mage_ai.services.spark.config import SparkConfig
+from typing import List
 import os
 
 
+def get_file_names(jars: List) -> List:
+    '''
```

Comment:
Could you use the google docstring format?
Example: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/models/block/utils.py#L12-L21

Reply:
Done making the suggested change.